wasm/main.go

```diff
@@ -1,69 +1,720 @@
+								value := values.Field(key).Interface()
+								if (name == "MaxStorage") || (name == "UsedStorage") {
+									OutputArea := jsDoc.Call("getElementById", name)
+									if !OutputArea.Truthy() {
+										log.Error("Unable to get output text area in settings keys")
+										return
+									}
+									sValue := value
+									if name == "MaxStorage" {
+										sValue = fmt.Sprintf("%.2f %s", value, "GB")
+									}
+									if name == "UsedStorage" {
+										sValue = fmt.Sprintf("%.2f %s", value, "%")
+									}
+									OutputArea.Set("innerHTML", sValue)
+
+								}
+							}
+						}
+					default:
+						{
+							log.Debug("Default Hit")
+						}
+					}
+				} else {
+					log.Debug("Not Handled Yet")
+				}
+
+			}
+		}()
+		return nil
+	})
+	return jsonfunc
+}
+
+func GetData(payload map[string]interface{}, funcName string) []uint8 {
+	buf, err := json.Marshal(payload)
+	if err != nil {
+		log.Error("Error in marshalling payload in : ", funcName, err.Error())
+		return nil
+	}
+	resp, err := http.Post(GATEWAY, "application/json", bytes.NewReader(buf))
+	if err != nil {
+		log.Error("Error in getting response in : ", funcName, err.Error())
+		return nil
+	}
+	defer resp.Body.Close()
+	respBuf, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Error("Error in reading respBuf in : ", funcName, err.Error())
+		return nil
+	}
+	data := make(map[string]string)
+	err = json.Unmarshal(respBuf, &data)
+	if err != nil {
+		log.Error("Error in unmarshalling respbuf in : ", funcName, err.Error())
+		return nil
+	}
+
+	var out Out
+	err = json.Unmarshal([]byte(data["val"]), &out)
+	if err != nil {
+		log.Error("Error in unmarshalling data in : ", funcName, err.Error())
+		return nil
+	}
+
+	val, err := json.MarshalIndent(out.Data, "", " ")
+	if err != nil {
+		log.Error("Error in marshalling out in : ", funcName, err.Error())
+		return nil
+	}
+	return val
+}
+
+
+func SetDisplay(Id string, Attr string, value string){
+	jsDoc := js.Global().Get("document")
+	if !jsDoc.Truthy() {
+		log.Error("Unable to get document object in: ", Id)
+		return
+	}
+	OutputArea := jsDoc.Call("getElementById", Id)
+	if !OutputArea.Truthy() {
+		log.Error("Unable to get output area in: ", Id)
+		return
+	}
+	OutputArea.Set(Attr, value)
+}
+
+func CreateElement(Id string, element string, Attr string, value string) {
+	jsDoc := js.Global().Get("document")
+	if !jsDoc.Truthy() {
+		log.Error("Unable to get document object in: ", Id)
+		return
+	}
+	OutputArea := jsDoc.Call("createElement", element)
+	if !OutputArea.Truthy() {
+		log.Error("Unable to create div in: ", Id)
+		return
+	}
+	if value != "" {
+	OutputArea.Set(Attr, value)
+	}
+	jsDoc.Call("getElementById", Id).Call("appendChild", OutputArea)
+}
+
+func GetID() js.Func {
+	jsonFunc := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		go func() {
+			payload := map[string]interface{}{
+				"val": "hive-cli.exe%$#id%$#-j",
+			}
+			val := GetData(payload, "GetID")
+
```

Comment:
empty lines

Reply:
I'll get rid of all empty lines together